docs/source/cluster_management/clis.rst

```diff
@@ -18,6 +18,53 @@ The CLI now comes prebuilt as an SLS distribution to be used with any AtlasDB-ba
 You can find the versions `here <https://palantir.bintray.com/releases/com/palantir/atlasdb/atlasdb-cli-distribution/>`__ for download.
 You can use the CLI by unpacking the tar ball and running the executable in ``service/bin/atlasdb-cli``.
 
+CLI Options
+===========
```

Comment:
Thanks a lot for doing this!

nit: maybe worth clarifying as 'General CLI options' or something like that?